tests/resources/Harbor-Util.robot

```diff
@@ -627,3 +630,19 @@ Basic Docker Command With Harbor
     Log  ${output}
     Should Be Equal As Integers  ${rc}  0
     Should Contain  ${output}  Untagged
+
+Fetch Harbor Self Signed Cert
+    [Tags]  secret
+    [Arguments]  ${harbor-ip}  ${user}=admin  ${password}=%{TEST_PASSWORD}
+    ${rc}=  Run And Return Rc  wget -q --tries=10 --connect-timeout=10 --auth-no-challenge --no-check-certificate --user ${user} --password ${password} https://${harbor-ip}/api/systeminfo/${harbor_cert}
```

Comment:
it's sort of static because its value is given in ***Variables***. I make it a suite variable instead of hardcoding it to leave room for harbor changing api.